change_notes/2024-12-08-identifier-hiding.md

```diff
@@ -0,0 +1,7 @@
+ - `A2-10-1` - `IdentifierHiding.ql`:
+   - Improved evaluation performance.
+   - Addressed false negatives where nested loops used the same variable name.
+   - Exclude cases where a variable declared in a lambda expression shadowed a globa or namespace variable that did not appear in the same translation unit.
+ - `RULE-5-3` - `IdentifierHidingC.ql`:
```

Comment:
```suggestion
 - `RULE-5-3` - `IdentifierHiding.ql`:
```

Reply:
Oh, yeah, that's right, my mistake.
Thanks a lot for clarifying and my excuses for the noise 🙇🏻

Reply:
no problem, thanks for the review! :)